cmd/jimmsrv/main.go

```diff
@@ -141,19 +142,26 @@ func start(ctx context.Context, s *service.Service) error {
 		return errors.E("jimm session store secret must be at least 64 characters")
 	}
 
+	hostKeyRaw := os.Getenv("JIMM_SSH_HOST_KEY")
+	fingerprints, err := ssh.GetFingerprintsFromPrivateKey([]byte(hostKeyRaw))
```

Comment:
i think we should differentiate between "JIMM_SSH_HOST_KEY not specified" and "Cannot parse hostkey"?